pkgs/applications/misc/playonlinux/default.nix

```
@@ -127,6 +120,15 @@ in stdenv.mkDerivation {
     done
   '';
 
+  dontWrapGApps = true;
+  postFixup = ''
+    makeWrapper $out/share/playonlinux/playonlinux{,-wrapped} \
+      --prefix PATH : ${binpath} \
+      ''${gappsWrapperArgs[@]}
+    makeWrapper ${steam-run}/bin/steam-run $out/bin/playonlinux \
+      --add-flags $out/share/playonlinux/playonlinux-wrapped \
```

this doesn't look right
```suggestion
      --add-flags $out/share/playonlinux/playonlinux-wrapped
```